### PR TITLE
NET-668 Modify S7130: Add vbnet rspec

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by mary-georgiou-sonarsource — APPROVED** [review]
LGTM

**Comment by mary-georgiou-sonarsource** [comment]
Converted to DRAFT so we don't merge before the rule is merged.

File: rules/S7130/metadata.json [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+  "title": "First/Single should be used instead of FirstOrDefault/SingleOrDefault on collections that are known to be non-empty",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "1min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Major",
```
Comment (mary-georgiou-sonarsource) [inline rules/S7130/metadata.json:11]: re-seeing this - is it really major? 
Maybe medium?

Comment (pavel-mikula-sonarsource) [reply rules/S7130/metadata.json:11]: Severities are "Info","Minor","Major","Critical","Blocker". Major is maped to Medium,
https://docs.sonarsource.com/sonarqube/9.9/user-guide/issues/

It's closer to "unused parameter" and "why the hell is this here" than "this is just too long line and that makes it hard to read"

**Pull request merged**